Resources/public/js/views/serverside/ez-sectionserversideview.js

```diff
@@ -48,11 +49,32 @@ YUI.add('ez-sectionserversideview', function (Y) {
                     multiple: true,
                     data: {
                         sectionId: button.getAttribute('data-section-rest-id'),
-                        afterUpdateCallback: unsetLoading,
+                        afterUpdateCallback: assignSectionCallback,
                     },
                 },
             });
         },
+        
+        /**
+         * Assign section to contents callback. It sets notifications to done
+         * state and unsets loading state from assign button.
+         * 
+         * @method _assignSectionCallback
+         * @protected
+         * @param {Y.Node} button
+         */
+        _assignSectionCallback: function(button){
+            var unsetLoading = Y.bind(this._uiUnsetAssignSectionLoading, this, button);
```

Comment:
you don't need to do that here, you can directly call `_uiUnsetAssignSectionLoading` with:

```js
this._uiUnsetAssignSectionLoading(button);
```